src/rospkg/os_detect.py

```diff
@@ -215,6 +215,24 @@ def get_codename(self):
         raise OsNotDetected("called in incorrect OS")
 
 
+class OpenEmbedded(OsDetector):
+    """
+    Detect OpenEmbedded.
+    """
+    def is_os(self):
+        return "ROS_OS_OVERRIDE" in os.environ and os.environ["ROS_OS_OVERRIDE"] == "oe"
+
+    def get_version(self):
```

Comment:
There is a DISTRO_VERSION that could be checked for in /etc/issue. This might be valuable to pass through if it's available.

Reply:
Given that we're not supporting running rosdep on the OpenEmbedded target system initially, it'll be better to add it later on, when we can actually test it.

There are multiple possibilities for running rosdep for OpenEmbedded:
  1. Run it on the host machine that cross-compiled;
  2. Run it on the target device that was cross-compiled;
  3. Run it on the host machine that cross-compiled but connect to the target device over ssh...